modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java

```diff
@@ -150,7 +150,16 @@ public void processAuthenticationResponse(
 
 			List<String> emails = jwtClaimsSet.getStringListClaim("emails");
 
-			claims.put("email", emails.get(0));
+			if (emails == null || emails.isEmpty()) {
```

Comment:
Hey @tomibiro, use the `ListUtil#isEmpty` method instead and add tests to cover the issue. Thx